nodejs/BUILD

```diff
@@ -105,6 +108,61 @@ deploy_npm(
     release = deployment["npm.release"],
 )
 
+ts_project(
+    name = "typedoc_runner",
```

Comment:
Let's leave a comment here saying that the runner can't be in `tool/typedoc` because the path configurations did not work when placed below the modules being documented